tools/fw-update/rs-fw-update.cpp

```diff
@@ -119,6 +119,40 @@ void list_devices(rs2::context ctx)
     }
 }
 
+int write_to_mipi_device( const std::vector< uint8_t > & fw_image)
+{
+    // Write firmware to appropriate file descritptor
+    std::cout << std::endl << "Update can take up to 2 minutes" << std::endl;
+    std::ofstream fw_path_in_device( "/dev/d4xx-dfu504", std::ios::binary );
+    if( fw_path_in_device )
+    {
+        bool done = false;
+        std::thread t1(
+            [&done]()
+            {
+                for( int i = 0; i < 101 && ! done; ++i )
+                {
```

Comment:
Why 101? 😀

Reply:
Show percentage 0-100. Added a comment.

Reply:
So we finish on Done which is probably not 100 seconds right?
Maybe we should add a 100% print on done=true?

I will see in the video